css/_recording.scss

```diff
@@ -1,3 +1,41 @@
+/**
+ * The recording spinner style.
+ */
 .recordingSpinner {
     vertical-align: top;
 }
+
+/**
+ * The recording button popup style.
+ */
+.recording-popup-dialog {
+    width: 140px;
+
+    /**
+     * The popup item style.
+     */
+    .recording-popup-item {
+        font-size: 12px;
+        position: relative;
+        top: -5px;
```

Comment:
Instead of pixel pushing, would it be worthwhile to try to center it using other techniques? Usually I fall back to flexbox because it's what I'm used to but there might some other more suitable ways.